test/test_recommendations.py

```diff
@@ -40,13 +40,13 @@ def setUpClass(cls):
         app.debug = False
         # Set up the test database
         app.config['SQLALCHEMY_DATABASE_URI'] = DATABASE_URI
+        Recommendation.init_db(app)
```

Comment:
why we need to move init_db here?

Reply:
I saw someone do that in the forum to fix their issues with elephant sql.